### PR TITLE
Extract path mapping

### PR DESCRIPTION
**Pull request by jasony0206**

### CODE REVIEW

File: config_parser.cc [hunk 1]
```diff
@@ -41,6 +42,30 @@ std::string NginxConfig::getConfigValue(std::string configName) {
   return "";
 }
 
+// Extract path mapping from config
+std::unordered_map<std::string, std::string> NginxConfig::extract_path_map() {
+  std::unordered_map<std::string, std::string> map;
+
```
Comment (tincan24) [inline config_parser.cc]: Maybe explain what the code below does

File: server_config.h [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 #include "config_parser.h"
 
+// TODO: add tests for server config
```
Comment (tincan24) [inline server_config.h]: Should add some unit tests before merging

Comment (jasony0206) [inline server_config.h]: addressed!

**Comment by tincan24** [comment]
Nice changes. Looks good to me.

**Pull request merged**